blueocean-core-js/src/js/components/ContentPageHeader.jsx

```diff
@@ -26,7 +27,9 @@ export const SiteHeader = props => {
     return (
         <BasicHeader className="ContentPageHeader">
             <TopNav>
-                <BlueLogo href={props.homeURL} />
+                <Extensions.Renderer extensionPoint="jenkins.header.logo">
+                    <BlueLogo href={props.homeURL} />
```

Comment:
Well I guess there is no docu and this. I looked at the code to find out ;)

What happens here is that yes @batmat some kind of fallback but every extensionPoint implementation will get the the `<BlueLogo href={props.homeURL} />` as props.children passed to it. So it can decide what to do with it. In case there are no implementors the `Ext.Renderer` will render those children.